client/signup/config/steps.js

```diff
@@ -85,16 +85,7 @@ export default {
 		providesToken: true,
 		providesDependencies: [ 'bearer_token', 'username' ],
 		unstorableDependencies: [ 'bearer_token' ],
-	},
-
-	'user-social': {
```

Comment:
Why did we remove this step? It is still referenced from the [`social` flow](https://github.com/Automattic/wp-calypso/blob/69c7ab3742e5253aec0e6f19643d6877c7b1f0dd/client/signup/config/flows.js#L263).

Reply:
Or do we plan to remove [this flow](https://wordpress.com/start/social/user-social)?

Reply:
Good catch. Removed.

Reply:
This step is still referenced [here](https://github.com/Automattic/wp-calypso/blob/e30c670f16a492cdc5f718b533b41c7753e6cf0b/client/signup/config/step-components.js#L43).